src/module.c

```diff
@@ -1052,12 +1052,14 @@ void __attribute__((destructor)) RediSearch_CleanupModule(void) {
   }
   invoked = 1;
 
-  CursorList_Destroy(&RSCursors);
+  
 
   Indexes_Free(specDict_g);
   dictRelease(specDict_g);
   specDict_g = NULL;
 
+  CursorList_Destroy(&RSCursors);
```

Comment:
Please explain why this was change and add a comment if free order is important.